src/MailingListBundle/Command/SyncMailingListCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Opifer\MailingListBundle\Command;
+
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use AppBundle\Reservation\ReservationManager;
+use Opifer\MailingListBundle\Entity\MailingList;
+
+class SyncMailingListCommand extends ContainerAwareCommand
```

Comment:
I'd call this `SyncSubscriptionCommand`, since you're syncing the subscriptions here.